english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/teleport/scream/scream.md

```diff
@@ -0,0 +1,4 @@
+You manage to scream for the entire 11 seconds it takes to 
+fall to the ground. The little red fellow seems to be muttering
+something, but you can't hear anything over the sound of your
+own screaming. You don't notice when he disappears. Your broken body is soon burnt to a crisp beside the flaming tower. Better use your wits next time.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.